pkg/metricsstore/metricsstore_test.go

```diff
@@ -43,10 +43,40 @@ func TestK8sAPIEventCounter(t *testing.T) {
 
 		assert.Equal(http.StatusOK, rr.Code)
 
-		expectedResp := fmt.Sprintf(`# HELP osm_k8s_api_event_count This counter represents the number of events received from the Kubernetes API Server
+		expectedResp := fmt.Sprintf(`# HELP osm_k8s_api_event_count represents the number of events received from the Kubernetes API Server
 # TYPE osm_k8s_api_event_count counter
 osm_k8s_api_event_count %d
 `, i /* api event count */)
 		assert.Contains(rr.Body.String(), expectedResp)
 	}
 }
+
+func TestProxyConnectCount(t *testing.T) {
```

Comment:
This test is not too important. Eventually we will group metrics by their category (counter, guage etc.) for basic coverage.